cli/services/package_service/package_service.go

```diff
@@ -81,5 +99,14 @@ func (packageService PackageService) UninstallPackage(pkg models.Package, bbeCon
 			return nil
 		}
 	}
-	return fmt.Errorf("package %s not found", pkg.Name)
+	return fmt.Errorf("Package `%s` not found", pkg.Name)
+}
+
+func IsPackageInstalled(pkg models.Package) bool {
+	cmd := execCommand("helm", "status", pkg.Name, "--namespace", pkg.Name)
```

Comment:
We need to set `--kube-context` to make sure we use the BBE cluster context (if you have multiple clusters). See the Helm commands above

Reply:
Maybe we should do some sort of a wrapper function for the helm commands 🤷

Reply:
Let's do it, #21